rosbag2_py/src/rosbag2_py/_storage.cpp

```diff
@@ -225,7 +226,8 @@ PYBIND11_MODULE(_storage, m) {
           topics_with_message_count,
           compression_format,
           compression_mode,
-          custom_data
+          custom_data,
+          ros_distro,
         };
       }),
     pybind11::arg("version") = 6,
```

Comment:
@emersonknapp Just in case. It seems forgotten to increment version number Python code.

Reply:
@emersonknapp Please don't forget to increment default metadata version. 
i.e. pybind11::arg("version") = **7**,

Reply:
Done (incremented to 8 now)